services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -428,6 +437,76 @@ async def pull_input_ports(
     return transferred_bytes
 
 
+@containers_router.post(
+    "/containers/directory-watcher:disable",
```

Comment:
https://google.aip.dev/156

Reply:
So this would be a `PATCH /containers/directory-watcher` with an argument `is_enabled` which could be true or false?

Reply:
agreed on this